tools/scripts/precheck.mjs

```diff
@@ -1,7 +1,6 @@
 import { execSync } from 'child_process';
 import chalk from 'chalk';
-import devkit from '@nx/devkit';
-const { getPackageManagerCommand } = devkit;
+import { getPackageManagerCommand } from '@nx/devkit';
```

Comment:
Glad I got the tsconfig working properly. The default import and manual destruct hurt me to do.